ivadomed/utils.py

```diff
@@ -16,6 +27,7 @@
 FP_COLOUR = 2
 FN_COLOUR = 3
 
+AXIS_DCT = {'sagittal': 0, 'coronal': 1, 'axial': 2}
```

Comment:
already defined in `main.py`. Could be defined once (e.g. as a parent class that includes all useful dict/variables) and then import it